src/sqlfluff/rules/L001.py

```diff
@@ -1,11 +1,13 @@
 """Implementation of Rule L001."""
+from typing import List
 from sqlfluff.core.rules import BaseRule, LintResult, LintFix, RuleContext
-from sqlfluff.core.rules.crawlers import SegmentSeekerCrawler
+from sqlfluff.core.rules.crawlers import RootOnlyCrawler
 from sqlfluff.utils.functional import sp, FunctionalContext
 from sqlfluff.core.rules.doc_decorators import (
     document_fix_compatible,
     document_groups,
 )
+from ..utils.reflow.classes import ReflowSequence
```

Comment:
why not `sqlfluff.utils` ?

Reply:
That's my editor being "helpful" - you're right.